jobs/javascript/http/dapr.yaml

```diff
@@ -0,0 +1,13 @@
+version: 1
+apps:
+  - appID: job-service
+    appDirPath: ./job-service/
+    appPort: 6200
+    daprHTTPPort: 6280
+    schedulerHostAddress: localhost
```

Comment:
```suggestion
```

Reply:
We are in the process of removing this line from all the quickstarts Constantin.